ts/utils/url.ts

```diff
@@ -15,3 +19,21 @@ export function getResourceNameFromUrl(
   const resourceName = splitted[splitted.length - 1].toLowerCase();
   return includeExt ? resourceName : resourceName.split(".")[0];
 }
+
+/**
+ * Return the fucntion to:
+ * - copy the value, if valueType is COPY
+ * - navigate to the map, if valueType is MAP
+ * - navigate to a browser, if valueType is LINK
+ */
+
+export function handleItemOnPress(
+  value: string,
+  valueType?: "MAP" | "COPY" | "LINK"
+): () => void {
+  return valueType === "MAP"
+    ? () => openMaps(value)
+    : valueType === "COPY"
+      ? () => clipboardSetStringWithFeedback(value)
+      : () => Linking.openURL(value).then(() => 0, () => 0);
```

Comment:
switch is more readable and if we will add another case it should be easy to handle it
```typescript
switch (valueType) {
    case "MAP":
      return () => openMaps(value);
    case "COPY":
      return () => clipboardSetStringWithFeedback(value);
    default:
      return () => Linking.openURL(value).then(() => 0, () => 0);
  }
```

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1426/commits/e80eb87bd8dd3362a1a333104e0c477305b60aa3